include/alpaka/acc/Tag.hpp

```diff
@@ -69,4 +70,21 @@ namespace alpaka
         alpaka::TagFpgaSyclIntel,
         alpaka::TagGpuSyclIntel>;
 
+    //!  \brief Function to print the names of each tag in the given tuple of acc tags
+    //!  \tparam TTuple is the type of the tuple of tags
+    template<typename TTuple>
+    void printAccTags()
+    {
+        // Check if the tuple is empty using std::tuple_size
+        if(std::tuple_size<TTuple>{} == 0)
+        {
+            std::cout << "No Acc Tags!";
+        }
+        else
+        {
+            std::cout << "Acc Tags: ";
+            std::apply([](auto... args) { ((std::cout << args.get_name() << " "), ...); }, TTuple{});
```

Comment:
I suggest something like this following code (Not tested) to have a comma seperated list.

```C++
std::apply(
  [](auto... args) { 
    auto i = std::size_size_v<TTuple>;
    ((std::cout << args.get_name() << (--i > 0u ? "," : "")), ...);
  }, 
  TTuple{}
);

```

Reply:
ok, good idea

Reply:
alternatives exists using `make_index_sequence` but i preferred this since cleaner. thanks.